Various fixes (#70)
epinio/docs: src/howtos/install_epinio_on_rancher_desktop.md

```
Previous commands are for Bash, but can be easily adapted for Windows shell.

> If you do not want to use the default "magic" DNS please follow [DNS setup](dns_setup.md).

Reviewer: ```suggestion
If you don't want to use the default "magic" DNS, please follow [DNS setup](dns_setup.md).
```
